<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderCalculator</class>
 <widget class="QDialog" name="OrderCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Order Production Time Calculator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lbl_name">
     <property name="text">
      <string>Order Production Time Calculator</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_product">
       <property name="text">
        <string>Product:</string>
       </property>
       <property name="class" stdset="0">
        <string>form_lbl</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cb_product"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbl_machine">
       <property name="text">
        <string>Machine:</string>
       </property>
       <property name="class" stdset="0">
        <string>form_lbl</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cb_machine"/>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="sb_quantity">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbl_quantity">
       <property name="text">
        <string>Quantity to produce:</string>
       </property>
       <property name="class" stdset="0">
        <string>form_lbl</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btn_calculate">
     <property name="styleSheet">
      <string notr="true">background-color: #333333;
    color: white;
    border-radius: 4px;
    padding: 6px 12px;
    font-weight: bold;
    font-size: 13px;</string>
     </property>
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_efficiency" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lbl_machine_capacity">
        <property name="text">
         <string>Machine capacity:</string>
        </property>
        <property name="class" stdset="0">
         <string>form_lbl</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_capacity">
        <property name="text">
         <string>[XXX]</string>
        </property>
        <property name="class" stdset="0">
         <string>lbl_value</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblLocalStatus">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #3C4B6E;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>OrderCalculator</sender>
   <signal>accepted()</signal>
   <receiver>OrderCalculator</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
